pallets/stateful-storage/src/benchmarking.rs

```diff
@@ -47,6 +46,26 @@ benchmarks! {
 		assert!(page_result.data.len() > 0);
 	}
 
+	upsert_page {
```

Comment:
Nice. I am guessing your updating the weights on Jenkins and weighting for them to complete to wire this up?

Reply:
I'm running benchmarks locally to check, but I think we're holding off on Jenkins and weighting until just before we merge the whole `private-graph-main` branch to `main`

Reply:
It might not be a bad idea to generate this and check them since you already created the benchmark.